### ISSUE TITLE
Citation form element needs label

### ISSUE THREAD

**Issue by jmcglone** [mlibrary/cozy-sun-bear#82]
`<input data-slot="title" id="permalink" value="http://hdl.handle.net/2027/fulcrum.4t64gn22b" readonly="readonly" onclick="this.select(); document.execCommand('copy');" type="text">`

- [x] Add `<label for="permalink">permalink</label>`

- [x] aria-label attribute does not exist or is empty
- [x] aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty or not visible
- [x] Form element does not have an implicit (wrapped) <label>
- [x] Form element does not have an explicit <label>
- [x] Element has no title attribute or the title attribute is empty
